TurkishFilter/sections/general_extensions.txt

```diff
@@ -59,6 +59,8 @@ eskisehirekspres.net#%#document.cookie="modalads=yes; path=/;";
 ! https://github.com/AdguardTeam/AdguardFilters/issues/53926
 izlenti.net#%#document.cookie = "gosterilen_popup=1; path=/";
 izlenti.net#%#//scriptlet('remove-attr', 'href', '#video > a[href][target="_blank"]')
+! https://github.com/AdguardTeam/AdguardFilters/issues/62079
+torrent-oyun.com#%#document.cookie = "esl=1";
```

Comment:
Что это правило делает? И каким должен быть конечный адрес редиректа?

Reply:
Источник редиректа https://www.torrent-oyun.com/
Таргет редиректа http://gestyy.com/ewLnlq
После первого редиректа скрипт устанавливает куку esl=1, после этого редирект больше не производится.

Reply:
у меня в любом случае открывается https://www.onclickperformance.com/jump/next.php? и потом идёт блокировка редиректа, сколько бы не кликал

Reply:
```
function checkTarget(e) {
  if (!getCookie('esl')) {
    var e = e || window.event;
    var win = doOpen('http://gestyy.com/ewLnlq');
    setCookie('esl', 1, 1 * 60 * 60 * 100);
  }
}
```
На <html> висит такой онклик.

Reply:
непонятно, куда оно должно перекидывать. Может там и должна быть реклама. Подождём ответа автора задачи

Reply:
В ишью жалоба на принудительный редирект на URL http://gestyy.com/ewLnlq,
этот онклик — единственное на странице, что содержит такой URL.

Reply:
It seems that this rule works fine too:
`torrent-oyun.com#%#//scriptlet("set-constant", "puShown", "true")`

So maybe it would be better to add `torrent-oyun.com` to the current rule:
https://github.com/AdguardTeam/AdguardFilters/blob/9313c39957bf61ef9cb842b247964e4be24b287d/TurkishFilter/sections/general_extensions.txt#L8

Reply:
@stanislav-atr